customer/src/test/java/no/unit/nva/customer/NvaApplicationDomainHandlerTest.java

```diff
@@ -98,6 +100,10 @@ private CustomerDto newCustomerDto() {
         return customer;
     }
 
+    private Map<String, String> randomAlternativeNames() {
```

Comment:
There exists a randomLanguageCodeGenerator in nva-datamodel-java. I am not sure if I want to drag that dependency into this code-base.